controller/setting_controller.go

```diff
@@ -563,6 +567,57 @@ func (sc *SettingController) updatePriorityClass() error {
 	return nil
 }
 
+func (sc *SettingController) updateKubernetesClusterAutoscalerEnabled() error {
```

Comment:
Similar to PQ or Toleration settings, should we put the IM pod annotation update in this function?

Reply:
Done.